src/torznab.ts

```diff
@@ -193,30 +296,40 @@ export async function searchTorznab(
 			(entry) => entry.indexerId === indexer.id,
 		);
 		return (
-			!entry ||
-			((!excludeOlder || entry.firstSearched > nMsAgo(excludeOlder)) &&
-				(!excludeRecentSearch ||
-					entry.lastSearched < nMsAgo(excludeRecentSearch)))
+			indexerDoesSupportMediaType(
+				getMediaType(searchee),
+				JSON.parse(indexer.categories),
+			) &&
```

Comment:
@ShanaryS do we still need this check here then? I suppose it doesn't hurt…

Reply:
Yes this is where we decide which indexers get searched. The one in `filterTimestamps` is necessary so the excluding of timestamp works since we never use certain searchees on certain indexers.